testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md

```diff
@@ -0,0 +1,21 @@
+| Step 1 - Gem Setup Options               | 1.  Launch the Create a Gem experience<br>2.  Observe the options in Step 1                                                                                                                                    | Dialog displays options for:<br><br>*   Default Gem<br>*   Prebuilt Gem<br>*   Asset Gem<br>*   C++ Tool Gem<br>*   Python Tool Gem<br>*   Choose existing template                                                                                                                                                                                                                                                                                                                         |
+| Step 2 - Gem Details                     | 1.  Launch the Create a Gem experience<br>2.  Select a gem type/template<br>3.  Left click Next                                                                                                                | *   All fields can be edited<br>    *   Gem Display Name<br>    *   Gem System Name<br>    *   Gem Summary<br>    *   Requirements<br>    *   License<br>    *   License URL<br>    *   Origin<br>    *   Origin URL<br>    *   Global Gem Tags<br>    *   Optional Global Gem Tag<br>    *   Optional Global Gem Tag<br>    *   User-defined Gem Tags<br>    *   Gem Location<br>    *   Gem Icon Path<br>    *   Documentation URL<br>*   Global Gem Tags field populates with gem fields |
+| Gem Details - Global Gem Tag duplication | 1.  Launch the Create a Gem experience<br>2.  Select a gem type/template<br>3.  Left click Next<br>4.  Attempt to enter the same Global Gem Tag into 2 of the Global Gem Tag fields                            | *   An error displays notifying the user that duplicates cannot be used for global gem tags                                                                                                                                                                                                                                                                                                                                                                                                 |
+| Step 2 - Required Fields not satisfied   | 1.  Launch the Create a Gem experience<br>2.  Select a gem type/template<br>3.  Left click Next<br>4.  On step 2 page, click next without entering information into some of the required fields (marked by \*) | *   User is unable to progress without required fields and a visual indicator suggests what is missing<br>*   Required Fields:<br>    *   Gem Display Name<br>    *   Gem System Name<br>        *   Alpha numeric requirement<br>        *   Unique error for alpha numeric<br>    *   License<br>    *   Global Gem Tag                                                                                                                                                                   |
```

Comment:
Should a tags field be required? I think it's reasonable to have a gem with initially 0 tags to it, especially if users are starting out.